src/solvers.jl

```diff
@@ -573,17 +573,18 @@ function vartypes_without_fixed(m::Model)
     return colCats
 end
 
-function collect_expr!(m, tmprow, terms::AffExpr)
+function collect_expr!(m, tmprow, terms::AffExpr, checkowned=true)
```

Comment:
I'd also rename ``checkowned`` to ``ignore_not_owned`` or something similar. ``checkowned=false`` implies to me that terms not owned will be added, not ignored.